bedrock/base/templates/macros.html

```diff
@@ -377,6 +377,16 @@
 </header>{#-- /#fxosnav-header --#}
 {%- endmacro %}
 
+{% macro google_play_button(application='firefox', class_name='', extra_attributes=None, id='', include_rel=True, target_blank=False) -%}
```

Comment:
I think it's safe that we can assume `rel=external` can always be attributed on the Play Store badge link, so I don't think it needs to be a parameter here?